docker/api

```diff
@@ -0,0 +1,11 @@
+FROM ruby
```

Comment:
@jvehent depends on feature set and what parts of ruby are being used.  I've had some slow .0 releases of the ruby runtime.  Usually I wait for x.x.3 before I consider a new dot release ready for use.